osc/fetch.py

```diff
@@ -176,7 +176,7 @@ def move_package(self, tmpfile, destdir, pac_obj=None):
                     sys.exit(1)
                 canonname = pac_obj.binary
 
-        fullfilename = os.path.join(destdir, canonname)
+        fullfilename = os.path.join(destdir, canonname.decode("utf-8"))
```

Comment:
Can you please use the helper function `decode_it()` from util/helper.py?

Like this:
`fullfilename = os.path.join(destdir, decode_it(canonname))`

Reply:
sure :+1: wasn't aware that there's a helper. fixed now

Reply:
Thank you very much for this catch.